index.html

```diff
@@ -181,64 +188,68 @@ <h1>Multi-Post Stories</h1>
-        <img src="photo/logos/medium.jpg" alt="">
+        <div class="header-text">
+          <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        
+        <button class="about-myself-link" type="submit">LET'S CONNECT</button>
+
+        <div class="about-myself-icons">
+          <img src="photo/logos/github.jpg" alt="">
+          <img src="photo/navs/Linkedin-icon.jpg" alt="">
+          <img src="photo/logos/angellist.jpg" alt="">
+          <img src="photo/logos/twitter.jpg" alt="">
+          <img src="photo/logos/medium.jpg" alt="">
+        </div>
+        
+        <button class="resume" type="submit">Get my resume</button> 
       </div>
-      
-      <button class="resume" type="submit">Get my resume</button> 
 
-      <div class="language">
-        <button class="language" type="submit">Languages</button>
+      <div class="language-wrapper">
+        <div class="language">
+          <button class="language" type="submit">Languages</button>
 
-        <div>
-          <img src="photo/ben/Dropdown.png" alt="">
+          <div>
+            <img src="photo/ben/Dropdown.png" alt="">
+          </div>
         </div>
-      </div>
 
-      <ul class="about-myself-ul">
-        <li class="java">  
-          <img src="photo/logos/js-image.png" alt="javascript">
-          <span>JavaScript</span>
-        </li>
-
-        <li class="html">
-          <img src="photo/logos/html-image.png" alt="html">
-          <span>HTML</span>
-        </li>
-
-        <li class="css">  
-          <img src="photo/logos/css-image.png" alt="css">
-          <span>CSS</span>
-        </li>
-      </ul>
-
-      <ul class="frameworks">
-        <li>
-          <h3>Frameworks</h3>
-          <img src="photo/ben/capture-side.PNG" alt="">
-        </li>
-      </ul>
-
-      <ul class="skills">
-        <li>
-          <h3>Skills</h3>
-          <img src="photo/ben/capture-side.PNG" alt="">
-        </li>
-      </ul>
+        <ul class="about-myself-ul">
+          <li class="java">  
+            <img src="photo/logos/js-image.png" alt="javascript">
+            <span>JavaScript</span>
+          </li>
+
+          <li class="html">
+            <img src="photo/logos/html-image.png" alt="html">
+            <span>HTML</span>
+          </li>
+
+          <li class="css">  
+            <img src="photo/logos/css-image.png" alt="css">
+            <span>CSS</span>
+          </li>
+        </ul>
+
+        <ul class="frameworks">
+          <li>
+            <h3>Frameworks</h3>
+            <img src="photo/ben/capture-side.PNG" alt="">
+          </li>
+        </ul>
+
+        <ul class="skills">
+          <li>
+            <h3>Skills</h3>
+            <img src="photo/ben/capture-side.PNG" alt="">
+          </li>
+        </ul>
+      </div>
     </section>
 
```

Comment:
- Please work on the horizontal scroll on the contact section so it will be removed from the webpage.
- Please use the background image as it is on the Figma design
- Please use the proper font-size as well so your design looks similar that of the Figma design.

##### Your design
![Screenshot from 2022-07-16 09-40-15](https://user-images.githubusercontent.com/58771507/179347584-fc40b98f-271d-4914-94d3-f82ecf9cd2ce.png)

##### Figma design

![Screenshot from 2022-07-16 09-40-48](https://user-images.githubusercontent.com/58771507/179347608-6a0a655b-f79f-42a2-bf97-f996031d4cbc.png)